src/main/java/org/hydev/mcpm/client/arguments/parsers/RefreshParser.java

```diff
@@ -0,0 +1,49 @@
+package org.hydev.mcpm.client.arguments.parsers;
+
+import net.sourceforge.argparse4j.inf.Namespace;
+import net.sourceforge.argparse4j.inf.Subparser;
+import org.hydev.mcpm.client.commands.entries.RefreshController;
+
+import java.io.IOException;
+import java.util.function.Consumer;
+
+/**
+ * Parser for the refresh command
+ *
+ * @author Azalea (https://github.com/hykilpikonna)
+ * @since 2022-11-24
+ */
+public record RefreshParser(RefreshController controller) implements CommandParser
```

Comment:
Wouldn't mind using some more specific names at this point. `Refresh`... what? I have to run the help command to find out? :(